frontend/src/components/ProgressIndicator/ProgressIndicator.tsx

```diff
@@ -1,35 +1,55 @@
 import { useMemo } from 'react'
-import { Box } from '@chakra-ui/react'
+import { Box, BoxProps } from '@chakra-ui/react'
+import { getColor } from '@chakra-ui/theme-tools'
 
+import { theme } from '~theme/index'
 import { MotionBox } from '~components/motion'
 
-const ActiveIndicator = (): JSX.Element => (
+const ActiveIndicator = ({ ...props }: BoxProps): JSX.Element => (
   <Box
+    // Top required to align it with CircleIndicators
+    top="0.125rem"
     width="1.5rem"
     height="0.5rem"
     borderRadius="full"
     backgroundColor="secondary.500"
+    position="absolute"
+    as="button"
```

Comment:
this is creating an extra tabbable button in the component which i don't think is intended.

my screen reader (voiceover) is also announcing this button.